### ISSUE TITLE
[TSDoc> Playground]: 'Selected' tab item such as 'DOM' is not accessible with keyboard on renavigation under 'Playground'.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/rushstack-websites#150]
**Test Environment:**
OS Version: 22H2 (OS Build 25370.1)
Browser: Edge Dev (Version 115.0.1866.1 (Official build) dev (64-bit))
URL: https://tsdoc.org/
Screen reader: Narrator

**Repro Steps:**

1. Open the above URL in edge dev browser and login with v-.
2. Turn on the narrator through Ctrl+ windows+ Enter key.
3. Navigate to the 'Playground' link and activate it.
4. Navigate to the 'DOM'tab item and activate it.
5. Now, renavigate on the 'DOM' tab item.
6. Observe the issue.

**Actual Result:**
On renavigation, selected' tab item such as 'DOM' is not accessible with keyboard.

**Expected Result:**
'Selected' tab item such as 'DOM' should be accessible with keyboard on renavigation under 'Playground'.
Screen reader should announce the state information along with the position such as **'DOM Tab item selected 2 of 5'.**

**User Impact:**
Keyboard dependent and screen reader user will get impacted as user will not not be able to renavigate back to the selected control and will not be able to make track of the content properly.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"
<img width="960" alt="2   Screen reader is not announcing the disabled state nor does it announce position for the list item under ‘Direct access’ section" src="https://github.com/microsoft/rushstack-websites/assets/104483005/e474c805-bd84-409d-bc5c-bdb414046f68">

https://github.com/microsoft/rushstack-websites/assets/104483005/c78d31c6-260e-4f23-ba99-5d61eeb3b2cf

**Comment by vaishali1397** [comment]
Closing this issue and will track with bug id: https://github.com/microsoft/rushstack-websites/issues/171
